editor/project_manager.cpp

```diff
@@ -376,20 +376,33 @@ class ProjectDialog : public ConfirmationDialog {
 		fdialog_install->popup_file_dialog();
 	}
 
-	void _create_folder() {
-		const String project_name_no_edges = project_name->get_text().strip_edges();
+	void _create_new_project_folder() {
+		_create_folder(project_name->get_text(), project_path);
+	}
+
+	void _create_install_folder() {
+		Vector<String> strings = project_path->get_text().split("/");
+		String filename = strings[strings.size() - 1];
+		String filename_no_extension = filename.trim_suffix("." + filename.get_extension());
```

Comment:
I added the `"." +` because `get_extension()` doesn't return the "." of the extension.

Reply:
`filename.get_basename()` is better in this case, although note that I have a PR that supersedes this